src/Analysis/Ast/Impl/Diagnostics/ErrorCodes.cs

```diff
@@ -29,5 +29,6 @@ public static class ErrorCodes {
         public const string ReturnInInit = "return-in-init";
         public const string TypingNewTypeArguments = "typing-newtype-arguments";
         public const string TypingGenericArguments = "typing-generic-arguments";
+        public const string WrongNumberArguments = "wrong-number-arguments";
```

Comment:
I don't really like this naming. I'd feel better if it were `wrong-argument-count` or similar.

Reply:
Sounds good.